### PR TITLE
Added new filters to web-bluetooth device-info

### PR DESCRIPTION
**Pull request by beaufortfrancois**
R=@jeffposnick, @g-ortuno

### CODE REVIEW

**Comment by jeffposnick** [comment]
:+1: from a code perspective. Unless @g-ortuno has any feedback from the API perspective, feel free to merge.

File: web-bluetooth/device-info.js [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
   if (filterService.startsWith('0x')) {
     filterService = parseInt(filterService, 16);
   }
+  if (filterService) {
+    options.filters.push({services: [ filterService ]});
+  }
+
+  let filterName = document.getElementById('name').value;
+  if (filterName) {
+    options.filters.push({name: name});
```
Comment (g-ortuno) [inline web-bluetooth/device-info.js]: Shouldn't this be:

```
{name: filterName}
```

?

**Comment by g-ortuno** [comment]
One small comment. Otherwise lgtm

**Comment by beaufortfrancois** [comment]
@g-ortuno Can you just simply look at me and slap me please in the face? Thank you!

**Comment by g-ortuno** [comment]
:) does name filtering work now?

**Comment by beaufortfrancois** [comment]
It does ;)

**Pull request merged**